server/src/main/java/org/opensearch/extensions/ExtensionRequest.java

```diff
@@ -26,54 +26,54 @@
  */
 public class ExtensionRequest extends TransportRequest {
```

Comment:
Can we have the generated protobuf class generate the transport request required methods while its at it?  I'd love to get rid of the error prone equals/hash/tostring also

Reply:
One step at a time :)
Definitely possible, some more framework to plumb in.

Reply:
https://stackoverflow.com/questions/41515779/unit-test-proto3-generated-objects-with-verify